### ISSUE TITLE
[Accessibility], WCAG 1.1.1. The Finto logo in the News section lacks alt text

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1966]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/en/

## Description of the problem
The Finto logo in the News section lacks alt text or decorative image markings. Screen readers find an image element, but do not get information about whether the image is important or only decorative.

Severity: Moderate
Priority: Medium

Fix: The publisher's logo can be marked as decorative (alt="") if the publisher's name is always written next to it

## Additional information (e.g. screenshots) about the problem

<img width="383" height="381" alt="Image" src="https://github.com/user-attachments/assets/cb1c113f-ee41-4d6b-8598-08487cf3d71d" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
The issue is not fixed yet.
